lmdeploy/pytorch/engine/engine.py

```diff
@@ -749,10 +750,8 @@ def _try_add_session(self, session_id: int):
 
     def stream_infer(self,
                      session_id: int,
-                     input_ids: List[int] = None,
-                     request_output_len: int = None,
-                     step: int = 0,
-                     sampling_param: SamplingParam = SamplingParam(),
+                     input_ids: List[int],
+                     gen_config: EngineGenerationConfig = None,
```

Comment:
According to documents. `gen_config` type is `GenerationConfig` instead of `EngineGenerationConfig`

Reply:
Engine can not perform tokenize

Reply:
OK